<ui version="4.0" >
 <author>Alvaro de Miguel</author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>AlbaranClienteListBase</class>
 <widget class="QWidget" name="AlbaranClienteListBase" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Albaranes a Clientes</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../bulmages-recursos-graficos.qrc" >:/Bulmafact48x48/images/png/48x48/i_bulmages_ventas_albaran_listado.48x48.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../../bulmages-recursos-graficos.qrc" >:/Bulmafact48x48/images/png/48x48/i_bulmages_ventas_albaran_listado.48x48.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Q3Frame" name="m_botonera" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QToolButton" name="mui_crear" >
          <property name="toolTip" >
           <string>Nuevo Albarán</string>
          </property>
          <property name="whatsThis" >
           <string>Nuevo Albarán</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages-recursos-graficos.qrc" >:/Genericos32x32/images/pendientes/i_fileNew.xpm</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_editar" >
          <property name="toolTip" >
           <string>Editar Albarán</string>
          </property>
          <property name="whatsThis" >
           <string>Editar Albarán</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages-recursos-graficos.qrc" >:/Genericos32x32/images/pendientes/i_edit.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_borrar" >
          <property name="toolTip" >
           <string>Borrar Albarán</string>
          </property>
          <property name="whatsThis" >
           <string>Borrar Albarán</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages-recursos-graficos.qrc" >:/Genericos32x32/images/pendientes/i_editDelete.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_imprimir" >
          <property name="toolTip" >
           <string>Imprimir Listado</string>
          </property>
          <property name="whatsThis" >
           <string>Imprimir Listado</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages-recursos-graficos.qrc" >:/Genericos32x32/images/pendientes/i_filePrint.xpm</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_filtrar" >
          <property name="toolTip" >
           <string>Filtrar Albaranes</string>
          </property>
          <property name="whatsThis" >
           <string>Filtrar Albaranes</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages-recursos-graficos.qrc" >:/Genericos32x32/images/pendientes/i_find.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_configurar" >
          <property name="toolTip" >
           <string>Configurar Listado</string>
          </property>
          <property name="whatsThis" >
           <string>Configurar Listado</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages-recursos-graficos.qrc" >:/Genericos32x32/images/pendientes/i_packageSettings.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_actualizar" >
          <property name="toolTip" >
           <string>Actualizar Listado</string>
          </property>
          <property name="whatsThis" >
           <string>Actualizar Listado</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages-recursos-graficos.qrc" >:/Genericos32x32/images/pendientes/i_reload.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>100</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1" >
          <property name="text" >
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_filtro" />
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="pixmapLabel1" >
     <property name="pixmap" >
      <pixmap/>
     </property>
     <property name="scaledContents" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Q3Frame" name="m_busqueda" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>5</number>
      </property>
      <property name="spacing" >
       <number>3</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>3</number>
        </property>
        <item>
         <widget class="QCheckBox" name="m_procesados" >
          <property name="text" >
           <string>Procesados</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>140</width>
            <height>31</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1_2_2" >
          <property name="text" >
           <string>Artículo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="BusquedaArticulo" name="m_articulo" />
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>60</width>
            <height>31</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1_2_2_2" >
          <property name="text" >
           <string>Cliente</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="BusquedaCliente" name="m_cliente" >
          <property name="minimumSize" >
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>3</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel1_4" >
          <property name="text" >
           <string>Fecha Inicial</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="BusquedaFecha" name="m_fechain" />
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>200</width>
            <height>21</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1_4_2" >
          <property name="text" >
           <string>Fecha Final</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="BusquedaFecha" name="m_fechafin" />
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>101</width>
            <height>21</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Q3Frame" name="m_configuracion" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>5</number>
      </property>
      <property name="spacing" >
       <number>3</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>3</number>
        </property>
        <item>
         <widget class="QCheckBox" name="mver_codigoalmacen" >
          <property name="text" >
           <string>&amp;Codigo Almacén</string>
          </property>
          <property name="shortcut" >
           <string>Alt+C</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mver_numalbaran" >
          <property name="text" >
           <string>&amp;Num. Albarán</string>
          </property>
          <property name="shortcut" >
           <string>Alt+N</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mver_fechaalbaran" >
          <property name="text" >
           <string>&amp;Fecha Albarán</string>
          </property>
          <property name="shortcut" >
           <string>Alt+F</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mver_nomcliente" >
          <property name="text" >
           <string>N&amp;om. Cliente</string>
          </property>
          <property name="shortcut" >
           <string>Alt+O</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>3</number>
        </property>
        <item>
         <widget class="QCheckBox" name="mver_idforma_pago" >
          <property name="text" >
           <string>&amp;Id. Forma Pago</string>
          </property>
          <property name="shortcut" >
           <string>Alt+I</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mver_descforma_pago" >
          <property name="text" >
           <string>Forma Pa&amp;go</string>
          </property>
          <property name="shortcut" >
           <string>Alt+G</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mver_numfactura" >
          <property name="text" >
           <string>Nu&amp;m. Factura</string>
          </property>
          <property name="shortcut" >
           <string>Alt+M</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mver_numnofactura" >
          <property name="text" >
           <string>Num. No Factura</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>3</number>
        </property>
        <item>
         <widget class="QCheckBox" name="mver_idusuario" >
          <property name="text" >
           <string>Id. &amp;Usuario</string>
          </property>
          <property name="shortcut" >
           <string>Alt+U</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mver_idcliente" >
          <property name="text" >
           <string>I&amp;d. Cliente</string>
          </property>
          <property name="shortcut" >
           <string>Alt+D</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mver_idalbaran" >
          <property name="text" >
           <string>Id. Albar&amp;án</string>
          </property>
          <property name="shortcut" >
           <string>Alt+Á</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mver_comentalbaran" >
          <property name="text" >
           <string>Comen&amp;tarios</string>
          </property>
          <property name="shortcut" >
           <string>Alt+T</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>3</number>
        </property>
        <item>
         <widget class="QCheckBox" name="mver_idalmacen" >
          <property name="text" >
           <string>Id. Almac&amp;én</string>
          </property>
          <property name="shortcut" >
           <string>Alt+É</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mver_totalalbaran" >
          <property name="text" >
           <string>Total Albaran</string>
          </property>
          <property name="shortcut" >
           <string/>
          </property>
          <property name="checked" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mver_totalbaseimp" >
          <property name="text" >
           <string>Total &amp;Base Imponible</string>
          </property>
          <property name="shortcut" >
           <string>Alt+B</string>
          </property>
          <property name="checked" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mver_totalimpuestos" >
          <property name="text" >
           <string>Total Impuestos</string>
          </property>
          <property name="shortcut" >
           <string/>
          </property>
          <property name="checked" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>3</number>
        </property>
        <item>
         <widget class="QCheckBox" name="mver_refalbaran" >
          <property name="text" >
           <string>Referencia</string>
          </property>
          <property name="shortcut" >
           <string/>
          </property>
          <property name="checked" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>350</width>
            <height>21</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="AlbaranClienteListSubform" name="mui_list" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>3</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>340</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="textLabel1_3" >
       <property name="text" >
        <string>Total</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_total" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="font" >
        <font>
         <family>Sans Serif</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="5" />
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>AlbaranClienteListSubform</class>
   <extends>QTableWidget</extends>
   <header>albaranclientelist.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>BusquedaCliente</class>
   <extends></extends>
   <header>busquedacliente.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>BusquedaArticulo</class>
   <extends></extends>
   <header>busquedaarticulo.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>BusquedaFecha</class>
   <extends></extends>
   <header>../../../bulmages/bulmalib/busquedafecha.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends></extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../bulmages-recursos-graficos.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>mui_filtrar</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_busqueda</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>222</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel" >
     <x>69</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mui_configurar</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_configuracion</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>255</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel" >
     <x>178</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
